app/MMB/src/navigation/MMBScreen.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Translation } from '@kiwicom/mobile-localization';
+import type { NavigationType } from '@kiwicom/mobile-navigation';
+import { HeaderBackButton } from 'react-navigation';
+
+const isDevelopment = process.env.NODE_ENV;
```

Comment:
Just a silly question - `NODE_ENV` is defined only during development and therefore it's safe to use it like this?

Reply:
It should be `development` in dev-mode and `production` when deployed. 

This is just a silly developer-mistake 🙈